next/components/forms/segments/AccountSections/TaxesFeesSection/ContactInformation.tsx

```diff
@@ -60,12 +61,16 @@ const ContactInformationSection = (props: any) => {
               data={{
                 label: t('correspondence_address'),
                 value:
-                  userData && userData.address
-                    ? `${userData.address.street_address}, ${postal_code_array
-                        ?.slice(0, 3)
-                        .join('')} ${postal_code_array?.slice(3).join('')} ${
-                        userData.address.locality
-                      }`
+                  userData &&
+                  (userData.address.street_address ||
+                    userData.address.postal_code ||
+                    userData.address.locality)
+                    ? `${
+                        userData.address.street_address &&
+                        (postal_code_array || userData.address.locality)
+                          ? `${userData.address.street_address},`
+                          : userData.address.street_address
+                      } ${postalCodeFormat(postal_code_array)} ${userData.address.locality}`
```

Comment:
It seems complicated but i need to solve render of comma symbol only when i have a street and something else